README.md

```diff
@@ -9,10 +9,26 @@ See the [HTML spec](https://html.spec.whatwg.org/multipage/forms.html#the-dialog
 
 ### Installation
 
-You may optionally install via NPM or Bower-
+You may optionally install via NPM -
 
     $ npm install dialog-polyfill
-    $ bower install dialog-polyfill
+
+
+There are 3 ways that you could include the dialog polyfill
+
+* include `dialog-polyfill.js` script directly in your HTML, which exposes a global `dialogPolyfill` function.
+* `import` (es modules)
+* `require` (commonjs/node)
+
+
+```javascript
+import dialogPolyfill from 'dialog-polyfill'  // modern es modules approach
```

Comment:
I'm not entirely a fan of this as it implies that you have extra loaders. I'd rather be explicit:

```js
import dialogPolyfill from './node_modules/dialog-polyfill/index.js';
// or
import dialogPolyfill from 'dialog-polyfill';  // for bundlers
```

But that's my personal opinion.